### PR TITLE
Bing search for China users

### PR DESCRIPTION
**Pull request by xcudder**
Adds a bing search as a fallback for users behind the GFW

### CODE REVIEW

**Comment by xcudder** [comment]
Just commenting to update the certificate status

**Comment by dankohn** [comment]
This is designed to close https://github.com/kubernetes/website/issues/9635

File: static/js/search.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+    window.renderGoogleSearchResults = () => {
```
Comment (zparnold) [inline static/js/search.js]: I know Google is blocked, so we may also want to change this function name to remove doubt for contributors in the future

**Comment by dankohn** [comment]
Cc @chenopis @zacharysarah @kbarnard10

**Comment by bradtopol** [comment]
I believe this was discussed in the last sig docs meeting by @zparnold.  I am assigning him to this PR as he gathered input from the SIG docs team regarding best direction for this PR.  cc @zacharysarah

**Comment by zacharysarah** [comment]
Because this is a pressing concern, let's merge this now and iterate on the dropdown we agreed was the better solution.

@hanjiayao Can you or a user on your team verify that search works, please? Thanks in advance. 😄 🙇 

/cc @mistyhacks @zparnold 

/lgtm
/approve

**Comment by dankohn** [comment]
Our colleague Tally in Beijing confirmed that this works correctly!

image

**Comment by dims** [comment]
Hello from Xi'an (travelling this week),  I can conform the Bing search works well too just like the screen shot above.

A couple of urls end up with 404, please take a look

image

- `https://survey.g.doubleclick.net/async_survey?site=6ip6epxiqjqn4qgzemchgtfziy`
- `https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/docs/search.md?pixel`

**Comment by zacharysarah** [comment]
@dims Those URLs are for Google site analytics, so it’s unsurprising that they’re 404. @chenopis, any thoughts about what or whether to take action?

**Comment by dankohn** [comment]
Our understanding is that Google Analytics works in China if you turn off a couple advanced features: https://grizzlypandamarketing.com/use-google-analytics-china/

@kbarnard10 Could you please work with Nate to do so on the kubernetes.io Google Analytics account?

**Comment by xiangpengzhao** [comment]
@dims just know you were in Xi'an. Missed a chance to meet you :(

**Comment by dims** [comment]
Ah dang! sorry @xiangpengzhao next time i will ping you.

**Comment by sftim** [comment]
See https://github.com/kubernetes/website/issues/37451

**Pull request merged**